### PR TITLE
Add Homeworld 3.

### PR DESCRIPTION
**Pull request by MetaRossi**
Add the Steam game Homeworld 3. Works really well. Possibly due to being an Unreal Engine 4 game.

### CODE REVIEW

**Review by devium335 — CHANGES_REQUESTED** [review]
Make sure to add your game to `/src/game-support/README.md`

**Comment by JoshuaBrest** [comment]
> Make sure to add your game to `/src/game-support/README.md`

Please ignore this comment

File: src/game-support/hw-3.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Homeworld 3
+<!-- script:Aliases [] -->
+
+{{#template ../templates/rating.md status=Platinum installs=Yes opens=Yes}}
```
Comment (JoshuaBrest) [inline src/game-support/hw-3.md:4]: I prefer a Gold review, but this could be considered platinum.

**Review by JoshuaBrest — APPROVED** [review]
LGTM ❤️

**Review by devium335 — APPROVED** [review]
My bad I forgot we migrated away from storing the games in that file

**Comment by JoshuaBrest** [comment]
> My bad I forgot we migrated away from storing the games in that file

All good!!!!! (:

**Review by IsaacMarovitz — APPROVED** [review]

**Pull request merged**